src/reference-manual/statements.qmd

```diff
@@ -1560,3 +1562,16 @@ is negligible or constant, we can't sample from that truncated
 posterior efficiently.  Programs need to use one-to-one mappings that
 guarantee the constraints are satisfied and only use reject statements
 to raise errors or help with debugging.
+
+## Fatal error statements
+
+The Stan `fatal_error` statement provides a mechanism to
+report errors or problematic values encountered during program
+execution and uniformly halt processing.
+
+Like the `print` or `reject` statement, the fatal_error statement accepts
+any number of quoted string literals or Stan expressions as arguments.
+
+It may be used to signal an unrecoverable error in blocks
```

Comment:
It -> The fatal error statement

(It's just generally bad form to start a paragraph with a pronoun, because they are hard to resolve.)